### PR TITLE
Use confluent-kafka. More options. Full tests coverage.

### PR DESCRIPTION
**Pull request by anguenot**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,27 +1,19 @@
...
-
-# command to install dependencies, e.g. pip install -r requirements.txt --use-mirrors
-install: pip install -U tox
-
-# command to run tests, e.g. python setup.py test
-script: tox -e $TOX_ENV
+sudo: required
+dist: trusty
+python:
+  - "2.7"
```
Comment (bsnyder788) [inline .travis.yml]: no support for 3.3/4?

File: tests/test_util.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+                    'action': 'drop', 'destination_port': -1, 'out': '',
+                    'proc': '0x00', 'id': '65299', 'dest_ip': '10.11.12.181'}
+
+        d1 = ast.literal_eval(str(expected))
+        self.assertDictEqual(d1, msg_s)
+
+    def test_date_str_to_ts(self):
+        date_str = 'Jun 22 12:49:16'
+        date_str_to_timestamp(date_str)
+        # FIXME will break next year
```
Comment (bsnyder788) [inline tests/test_util.py]: mean to leave this FIXME in here?

File: .travis.yml (hunk 1 above)
Comment (anguenot) [inline .travis.yml]: `syslog-ng` does not actually support Python 3 yet.

File: tests/test_util.py (hunk 2 above)
Comment (anguenot) [inline tests/test_util.py]: old one but will fix in that branch as well.

**Pull request merged**